src/components/Search/SearchFiltersParticipantsSelector.tsx

```diff
@@ -104,19 +104,20 @@ function SearchFiltersParticipantsSelector({initialAccountIDs, onFiltersUpdate}:
             true,
         );
 
-        const isCurrentUserSelected = selectedOptions.find((option) => option.accountID === chatOptions.currentUserOption?.accountID);
+        const currentUser = formattedResults.section.data.find((option) => option.accountID === chatOptions.currentUserOption?.accountID);
```

Comment:
I'd rename this variable, because there is a snippet below that confuses me, in lines 111-112:
```
 if (!currentUser) {
                chatOptions.currentUserOption.text = formattedName;
```

"if NOT current user, run some code on chatOptions.currentUserOption" - which sounds kinda confusing.